homeassistant/components/repetier/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "repetier",
   "name": "Repetier-Server",
   "documentation": "https://www.home-assistant.io/integrations/repetier",
-  "requirements": ["pyrepetier==3.0.5"],
+  "requirements": ["pyrepetierng==0.1.0"],
   "codeowners": ["@MTrab"],
```

Comment:
It would be good if you want to add yourself as codeowner since you're maintaining the connected library.

Reply:
okay, done. :)